src/components/ThemeSwitcher/context/SwitchThemeContext.tsx

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable react-refresh/only-export-components */
+
+import React, {
+  createContext,
+  useState,
+  useEffect,
+  type ReactNode,
+} from "react";
+
+interface ThemeContextProps {
+  theme: string;
```

Comment:
Здесь лучше явно указать тип темы, не просто `string`

```suggestion
  theme: Theme;
```